assets/js/admin/orders/order-overview/_refund.js

```diff
@@ -221,7 +221,7 @@ function recalculateRefundTotal() {
 $(document.body).on( 'click', '#edd-submit-refund-submit', function(e) {
 	e.preventDefault();
 	$('.edd-submit-refund-message').removeClass('success').removeClass('fail');
-	$( this ).attr( 'disabled', false ).addClass( 'updating-message' );
+	$( this ).attr( 'disabled', true );
```

Comment:
I know I approved this but I'm watching a migration in another screen and I like the `updating-message` icon behavior. I'm wondering what you would think of adding that back in, but removing the `button-primary` class from `$( this )` at the same time? That would get the spinner in the button, but the color would be the red spinner, which does work with the disabled button.

Reply:
Oh, I like that much more. Good suggestion.